catalog/core/catalog-core-urlresourcereader/src/main/java/ddf/catalog/resource/impl/URLResourceReader.java

```diff
@@ -306,6 +303,26 @@ public ResourceResponse retrieveResource(URI resourceURI, Map<String, Serializab
     }
   }
 
+  private URI setQualifierParamForUri(URI resourceURI, String qualifier)
```

Comment:
✏️ This might be a little clearer if it was just a void method that set the qualifier on the `resourceURI` argument. Or, maybe renaming the method to something like `createUriWithQualifier` could work too.

Reply:
The URI class provides no way for me to modify the URI otherwise I would just append the qualifier. I think the name works here though, createUriWithQualifier might sound like we're making an entirely different URI.

Reply:
I am a fan of using better verbs over `get` or `set` since they have a meaning for Java beans.  It is also misleading with this function since nothing is actually set in the parent class and could be static.